src/diffusers/schedulers/scheduling_pndm.py

```diff
@@ -113,6 +113,7 @@ def step(
         model_output: Union[torch.FloatTensor, np.ndarray],
         timestep: int,
         sample: Union[torch.FloatTensor, np.ndarray],
+        **kwargs,
```

Comment:
@patil-suraj @anton-l - I'd prefer a bit to keep the schedulers as clean as possible and instead make the pipeline more hacky.

Could we maybe delete the unused `kwargs` here and instead make use of  `inspect.signature` in the pipeline?
Analogs to https://github.com/huggingface/transformers/blob/05d3a43c59dadf815fe46df8fed8cc8e816c8588/src/transformers/generation_utils.py#L1171

Note that the schedulers should eventually work for all frameworks and it's IMO a components where we strongly favor readability and intuitive design over hacky/easy-to-use.

What do you think?

Reply:
I see. I didn't think of using `inspect.signature`. I am also not in full favor of adding unused arguments, but didn't find a better solution so went with it as I wanted to get it rolling today. 

Will remove this and adapt the pipeline.